### PR TITLE
Added setter for logstash prefixSeparator and dateformat

### PR DESCRIPTION
**Pull request by rgarcia89**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://developercertificate.org) signed
- [ ] Chart Version bumped (if the pr is an update to an existing chart)
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 9.3.2
+version: 9.3.3
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml:3]: This is a new feature so Version should be 9.4.0

File: charts/fluentd-elasticsearch/values.yaml [hunk 2]
```diff
@@ -68,6 +68,8 @@ elasticsearch:
   logstash:
     enabled: true
     prefix: "logstash"
+    # prefixSeparator: "-"
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml:71]: Please don't uncomment but use the currently used default values there.

File: charts/fluentd-elasticsearch/Chart.yaml (hunk 1 above)
Comment (rgarcia89) [reply charts/fluentd-elasticsearch/Chart.yaml:3]: done

**Comment by rgarcia89** [comment]
somehow github is not seeing my latest commits. I will close and reopen the PR

**Pull request closed without merging**